### PR TITLE
Add appendContent function

### PR DESCRIPTION
**Pull request by Caruki**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import './app.scss';
 import './components/search.scss';
 import { createElement } from './lib/dom';
 import { createTitle } from './components/title';
-import { createSearchInput } from './components/search';
-import { createSearchResults } from './components/search';
+import { createSearchInput, createSearchResults } from './components/search';
+import { appendContent } from './lib/dom';
```
Comment (lmachens) [inline src/app.js]: You can combine line 3 and 6:
```
import { createElement, appendContent } from './lib/dom';
```

File: src/index.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import './index.scss';
 import { app } from './app';
+import { appendContent } from './lib/dom';
 
 const elements = app();
-elements.forEach(element => {
-  document.body.appendChild(element);
-});
+
 //Alternative: elements.forEach(document.body.appendChild);
```
Comment (lmachens) [inline src/index.js]: You don't need that comment anymore :). 
Or you move that comment to the `appendContent` function.

File: src/assets/logo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...d="m86.78125 173.558594c-4.792969 0-8.679688-3.882813-8.679688-8.675782v-17.355468c0-4.792969 3.886719-8.679688 8.679688-8.679688s8.675781 3.886719 8.675781 8.679688v17.355468c0 4.792969-3.882812 8.675782-8.675781 8.675782zm0 0" fill="#ed634c"/><path d="m52.066406 190.914062c-1.34375 0-2.667968-.3125-3.867187-.910156l-17.359375-8.679687c-4.289063-2.140625-6.027344-7.355469-3.886719-11.644531 2.144531-4.289063 7.355469-6.03125 11.644531-3.886719l17.359375 8.675781c3.605469 1.800781 5.5 5.84375 4.574219 9.765625-.929688 3.921875-4.433594 6.6875-8.464844 6.679687zm0 0" fill="#ed634c"/><path d="m416.542969 173.558594c-4.792969 0-8.679688-3.882813-8.679688-8.675782v-17.355468c0-4.792969 3.886719-8.679688 8.679688-8.679688s8.675781 3.886719 8.675781 8.679688v17.355468c0 4.792969-3.882812 8.675782-8.675781 8.675782zm0 0" fill="#ed634c"/><path d="m451.253906 190.914062c-4.03125.007813-7.535156-2.757812-8.460937-6.679687-.929688-3.921875.964843-7.964844 4.574219-9.765625l17.355468-8.675781c4.289063-2.144531 9.503906-.402344 11.644532 3.886719 2.144531 4.289062.402343 9.503906-3.886719 11.644531l-17.355469 8.679687c-1.203125.597656-2.527344.910156-3.871094.910156zm0 0" fill="#ed634c"/></svg>
```
Comment (lmachens) [inline src/assets/logo.svg:1]: Looks like this file is not related to the other changes.

**Review by lmachens — COMMENTED** [review]
Good job! 
Only small optimizations requested.

**Review by Lenkly — APPROVED** [review]
